### ISSUE TITLE
Only show quits/parts/joins of active users

### ISSUE THREAD

**Issue by KrzaQ** [erming/shout#445]
On busy channels there's a lot of traffic going on, even if there's little conversation. Please add an option to show joins/parts/quits/nick changes only of people who have recently (configurable) been active. It might even be feasible to backtrack and insert join information only after the person says something.

**Comment by JocelynDelalande** [comment]
@KrzaQ thanks for taking time to report :)

That seems a too much opinionated and "magic" feature to me, what do others think ?

**Comment by astorije** [comment]
I agree, this is a feature that will lead to confusion. I can foresee people opening issues because they see join/part messages appearing or disappearing for no apparent reason.

I would be fine to see that in an external plugin, when [that plugin system chimera gets composed](https://github.com/erming/shout-roadmap/pull/5/files#diff-04c6e90faac2675aa89e2176d2eec7d8R29), but not part of the core.

Sorry to suggest that, @KrzaQ, as I mostly thank you for your time and your feedback, but I would like to label this issue as `wontfix` and close it. However, I invite you to post this as a plugin idea on the roadmap repository and/or to give your ideas on a plugin system.

**Comment by KrzaQ** [comment]
Thanks for the reply. You should see freenode/##c++ for an example of an extremely join-busy channel.

I understand that it's much better suited for a plugin, so I'll eagerly await the time it's possible.

That being said, I disagree that it would lead to the confusion, if it was an optional feature. A popup  explaining what's happening could also be displayed until the user clicks "I understand what's happening and how to make it unhappen, should I want it".

**Comment by astorije** [comment]
That makes sense, @KrzaQ. I am closing this then and adding your idea as the list of plugin ideas in the roadmap. Feel free to stop there and open issues/PRs with ideas that cross your mind :)
